src/dataHandler.py

```diff
@@ -60,4 +60,162 @@ def send_all_data(self, sendingTo, destination):
             writer = csv.writer(f)
             writer.writerow(data)
             f.close()
+
+    #gravitational acceleration 
+    gravity = 9.80665
+
+    #gas constant
+    gas_const = 8.3144598
+
+    #earth atmospheric molar mass
+    atmo_molar_mass = 0.289644
+
+    #vapourization heat of water
+    h2o_vapour_heat = 2501000
+
+    #specific gas constant of dry air
+    specif_gas_const_dry = 287
+    
+    #specific gas constant of water vapour
+    specif_gas_const_h2o = 461.5
+    
+    #specific heat of dry air
+    specif_heat_dryair = 1003.5
+
+    #initialized altitude at launch pad
+    initialized_altitude = 274.1
+    # just an estimate, actual data will be collected on site
+
+    #previous calculated height change
+    prev_height_change = 0
+    # previous change of height to compensate for Null case
+
+    #lapse rate of the air
+    lapse_rate = 0.0098
+    # this is the standard dry air lapse rate, altered by initializeLapseRate()
+
+
+    def initializeLapseRate(moist, press, temp):
+        """
+        Takes moistness, pressure, and temperature readings at launch site to find lapse rate. 
+        This should not be run after the initialization phase
+
+        Paramaters:
+            moist: float, water vapour pressure
+            press: float, pressure reading
+            temp: float, temperature reading
+
+        Returns:
+            lapse_rate: float, lapse rate (... duh)
+        """
+        numerator_numerator = h2o_vapour_heat * moist
+        numerator_denomenator = temp * specif_gas_const_h2o * (press - moist)
+
+        numerator = gravity * (1 + (numerator_numerator/numerator_denomenator))
+
+        denomenator_numerator = h2o_vapour_heat**2 * specif_gas_const_dry * moist
+        denomenator_denomenator = (specif_gas_const_h2o * temp)**2 * (press - moist)
+
+        denomenator = specif_heat_dryair + (denomenator_numerator/denomenator_denomenator)
```

Comment:
i don't know if there's any better way to name these, but it's a little bit confusing to have "numerator_numerator" and "numerator_denominator " and just "numerator"